diff --git a/flexslider.admin.inc b/flexslider.admin.inc
index 5f71506..6d47d3e 100644
--- a/flexslider.admin.inc
+++ b/flexslider.admin.inc
@@ -179,6 +179,13 @@ function flexslider_option_elements($options = array()) {
     '#maxlength' => 255,
     '#default_value' => isset($options['move']) ? $options['move'] : _flexslider_optionset_defaults('move'),
   );
+  $form['animation_slideshow']['allowOneSlide'] = array(
+    '#type' => 'checkbox',
+    '#title' => t('Allow One Slide'),
+    '#description' => t('Whether or not to allow a slider comprised of a single slide <em>Requires FlexSlider Library 2.2+</em>.'),
+    '#default_value' => isset($options['allowOneSlide']) ? $options['allowOneSlide'] : _flexslider_optionset_defaults('allowOneSlide'),
+  );
+
 
   // Navigation and Control Settings
   $form['nav_controls'] = array(
diff --git a/flexslider.module b/flexslider.module
index 2d3f089..492f418 100755
--- a/flexslider.module
+++ b/flexslider.module
@@ -17,7 +17,7 @@ function flexslider_libraries_info() {
     'download url' => 'https://github.com/woothemes/FlexSlider',
     'version arguments' => array(
       'file' => 'jquery.flexslider-min.js',
-      // jQuery FlexSlider v2.1
+      // jQuery FlexSlider v2.2+
       'pattern' => '/jQuery FlexSlider v(\d+\.+\d+)/',
       'lines' => 2,
     ),
@@ -420,8 +420,8 @@ function _flexslider_optionset_defaults($key = NULL) {
     'selector' => '.slides > li',
     'easing' => 'swing',
     'direction' => 'horizontal',
-    'reverse' => FALSE,  // @todo verify data value
-    'smoothHeight' => FALSE, // @todo verify data value
+    'reverse' => FALSE,
+    'smoothHeight' => FALSE,
     'startAt' => 0,
     'animationSpeed' => 600,
     'initDelay' => 0,
@@ -439,40 +439,22 @@ function _flexslider_optionset_defaults($key = NULL) {
     'minItems' => 0,
     'maxItems' => 0,
     'move' => 0,
-    //'start' => '',
-    //'before' => '',
-    //'after' => '',
-    //'end' => '',
-    //'added' => '',
-    //'removed' => '',
-
-    // @todo verify the 1.x options are still valid
-    // v1.x options
     'animation' => 'fade',
-    //'animationDuration' => 6000, -- replaced by 'animationSpeed'
-    //'slidedirection' => 'horizontal', -- replaced by "direction"
     'slideshow' => TRUE,
     'slideshowSpeed' => 7000,
     'directionNav' => TRUE,
     'controlNav' => TRUE,
-    //'keyboardnav' => TRUE, --  replaced by 'keyboard'
-    //'mousewheel' => FALSE,
     'prevText' => t('Previous'),
     'nextText' => t('Next'),
     'pausePlay' => FALSE,
     'pauseText' => t('Pause'),
     'playText' => t('Play'),
     'randomize' => FALSE,
-    //'slidetostart' => 0, // integer value, not boolean --  replace by "startAt"
     'animationLoop' => TRUE,
     'pauseOnAction' => TRUE,
     'pauseOnHover' => FALSE,
-    //'controlscontainer' => '.flex-nav-container', -- updated in v2
     'manualControls' => '',
-    //'startCallback' => 'function() {}', -- replace by 'start'
-    //'beforeCallback' => 'function() {}', -- replaced by 'before'
-    //'afterCallback' => 'function() {}', -- replaced by 'after'
-    //'endCallback' => 'function() {}', -- replaced by 'end'
+    'allowOneSlide' => TRUE,
   );
 
   // Typecast the values
@@ -540,6 +522,7 @@ function _flexslider_typecast_optionset(&$options) {
         case 'animationLoop':
         case 'pauseOnAction':
         case 'pauseOnHover':
+        case 'allowOneSlide':
           $options[$key] = (boolean)$value;
           break;
       }
