diff --git a/flexslider_views/theme/flexslider_views.theme.inc b/flexslider_views/theme/flexslider_views.theme.inc
index 26af201..cd491aa 100644
--- a/flexslider_views/theme/flexslider_views.theme.inc
+++ b/flexslider_views/theme/flexslider_views.theme.inc
@@ -8,7 +8,9 @@
  */
 
 /**
- * Add a preprocessor to prepare data from FlexSlider Views
+ * Prepares variables for flexslider views templates.
+ *
+ * @see flexslider-views.tpl.php.
  */
 function template_process_flexslider_views(&$variables) {
   // Only run the preprocessor if it is a view
diff --git a/theme/flexslider.theme.inc b/theme/flexslider.theme.inc
index 0932f8f..6febcde 100755
--- a/theme/flexslider.theme.inc
+++ b/theme/flexslider.theme.inc
@@ -50,7 +50,9 @@ function theme_flexslider_list_item(&$variables) {
 }
 
 /**
- * Template preprocess handler for 'flexslider' theme.
+ * Prepares variables for flexslider templates.
+ *
+ * @see flexslider.tpl.php.
  */
 function template_process_flexslider(&$variables) {
 
@@ -116,7 +118,9 @@ function template_process_flexslider(&$variables) {
 }
 
 /**
- * Process function for flexslider_list_item
+ * Prepares variables for flexslider list templates.
+ *
+ * @see flexslider-list.tpl.php.
  */
 function template_process_flexslider_list(&$variables) {
   // Reset the list of attributes
@@ -128,7 +132,9 @@ function template_process_flexslider_list(&$variables) {
 }
 
 /**
- * Process function for flexslider_list_item
+ * Prepares variables for flexslider list item templates.
+ *
+ * @see flexslider-list-item.tpl.php.
  */
 function template_process_flexslider_list_item(&$variables) {
   // Reset the list of attributes
