diff --git a/flexslider_views/theme/flexslider_views.theme.inc b/flexslider_views/theme/flexslider_views.theme.inc
index 45d2b1f..26af201 100644
--- a/flexslider_views/theme/flexslider_views.theme.inc
+++ b/flexslider_views/theme/flexslider_views.theme.inc
@@ -10,18 +10,18 @@
 /**
  * Add a preprocessor to prepare data from FlexSlider Views
  */
-function template_process_flexslider_views(&$vars) {
+function template_process_flexslider_views(&$variables) {
   // Only run the preprocessor if it is a view
-  if (!empty($vars['view'])) {
+  if (!empty($variables['view'])) {
     // Prepare all data to be passed to theme_flexslider().
     $items = array();
-    $view = $vars['view'];
+    $view = $variables['view'];
     $settings = array();
-    $settings['optionset'] = $vars['options']['optionset'];
+    $settings['optionset'] = $variables['options']['optionset'];
 
     $row_plugin = $view->style_plugin->row_plugin;
 
-    foreach ($vars['rows'] as $id => $row) {
+    foreach ($variables['rows'] as $id => $row) {
       // Render the row into a slide
       $view->row_index = $id;
       $item['slide'] = $row_plugin->render($row);
@@ -30,13 +30,13 @@ function template_process_flexslider_views(&$vars) {
     }
 
     // Add the slide items to the variables
-    $vars['items'] = $items;
+    $variables['items'] = $items;
 
     // Add the attributes
-    $settings['attributes']['id'] = $vars['options']['id'];
+    $settings['attributes']['id'] = $variables['options']['id'];
 
     // Reset the options to the processed values
-    $vars['settings'] = $settings;
+    $variables['settings'] = $settings;
   }
-  return $vars;
+  return $variables;
 }
diff --git a/theme/flexslider.theme.inc b/theme/flexslider.theme.inc
index 36f58b0..0932f8f 100755
--- a/theme/flexslider.theme.inc
+++ b/theme/flexslider.theme.inc
@@ -10,12 +10,12 @@
 /**
  * Default theme implementation for flexslider_list
  */
-function theme_flexslider_list(&$vars) {
+function theme_flexslider_list(&$variables) {
   // Reference configuration variables
-  $optionset = &$vars['settings']['optionset'];
-  $items = &$vars['items'];
-  $attributes = &$vars['settings']['attributes'];
-  $type = &$vars['settings']['type'];
+  $optionset = &$variables['settings']['optionset'];
+  $items = &$variables['items'];
+  $attributes = &$variables['settings']['attributes'];
+  $type = &$variables['settings']['type'];
   $output = '';
 
   // Build the list
@@ -45,19 +45,19 @@ function theme_flexslider_list(&$vars) {
 /**
  * Default theme implementation for flexslider_list_item
  */
-function theme_flexslider_list_item(&$vars) {
-  return '<li' . drupal_attributes($vars['settings']['attributes']) . '>' . $vars['item'] . $vars['caption'] . "</li>\n";
+function theme_flexslider_list_item(&$variables) {
+  return '<li' . drupal_attributes($variables['settings']['attributes']) . '>' . $variables['item'] . $variables['caption'] . "</li>\n";
 }
 
 /**
  * Template preprocess handler for 'flexslider' theme.
  */
-function template_process_flexslider(&$vars) {
+function template_process_flexslider(&$variables) {
 
   // Reference configuration variables
-  $optionset = &$vars['settings']['optionset'];
-  $settings = &$vars['settings'];
-  $items = &$vars['items'];
+  $optionset = &$variables['settings']['optionset'];
+  $settings = &$variables['settings'];
+  $items = &$variables['items'];
 
   // Set the default container type
   if (empty($settings['type'])) {
@@ -112,15 +112,15 @@ function template_process_flexslider(&$vars) {
   $settings['attributes'] = $attributes;
 
   // Finally, add the configuration to the page
-  flexslider_add($vars['settings']['attributes']['id'], $vars['settings']['optionset']);
+  flexslider_add($variables['settings']['attributes']['id'], $variables['settings']['optionset']);
 }
 
 /**
  * Process function for flexslider_list_item
  */
-function template_process_flexslider_list(&$vars) {
+function template_process_flexslider_list(&$variables) {
   // Reset the list of attributes
-  $vars['settings']['attributes'] = array(
+  $variables['settings']['attributes'] = array(
     // @todo find a way to detect the outter container class if possible
     'class' => array('slides'),
   );
@@ -130,15 +130,15 @@ function template_process_flexslider_list(&$vars) {
 /**
  * Process function for flexslider_list_item
  */
-function template_process_flexslider_list_item(&$vars) {
+function template_process_flexslider_list_item(&$variables) {
   // Reset the list of attributes
-  $vars['settings']['attributes'] = array();
+  $variables['settings']['attributes'] = array();
 
   // Reference configuration variables
-  $item = &$vars['item'];
-  $settings = &$vars['settings'];
-  $caption = &$vars['caption'];
-  $attributes = &$vars['settings']['attributes'];
+  $item = &$variables['item'];
+  $settings = &$variables['settings'];
+  $caption = &$variables['caption'];
+  $attributes = &$variables['settings']['attributes'];
 
   // Generated thumbnail support
   if (isset($settings['optionset']->options['controlNav']) and $settings['optionset']->options['controlNav'] === "thumbnails") {
